### ISSUE TITLE
cluster boundaries not visible

### ISSUE THREAD

**Issue by gubuntu** [kartoza/LEDET_BIMS#148]
A bunch of blue boundaries draw before being covered by the basemap and at one stage I noticed some faint yellow boundaries. 

- [x] cluster boundaries must draw over base map and be made more visible
- [ ] ~cluster boundaries (and all other geojson layers) are styled by SLD or MapBox JSON (not Kartoza custom styles)~ follow up in another ticket.
- [x] styles are configurable through admin (this will be met by GeoNode integration so don't do anything about this now)

depends on #159
